src/test/scala/mesosphere/marathon/core/task/jobs/ExpungeOverdueLostTasksActorTest.scala

```diff
@@ -73,6 +74,24 @@ class ExpungeOverdueLostTasksActorTest extends MarathonSpec with GivenWhenThen w
     testProbe.receiveOne(3.seconds)
 
     And("one kill call is issued")
-    verify(stateOpProcessor, once).process(any)
+    verify(stateOpProcessor, once).process(TaskStateOp.ForceExpunge(unreachable.taskId))
+    noMoreInteractions(stateOpProcessor)
+  }
+
+  test("a unreachable task with less then 24 hours with no status update should not be killed") {
+    Given("two unreachable tasks, one overdue")
+    val unreachable1 = MarathonTestHelper.minimalUnreachableTask("/unreachable1".toPath, since = Timestamp.apply(0))
+    val unreachable2 = MarathonTestHelper.minimalUnreachableTask("/unreachable2".toPath, since = Timestamp.now())
+
+    taskTracker.tasksByApp()(any[ExecutionContext]) returns Future.successful(TasksByApp.forTasks(unreachable1, unreachable2))
+
+    When("a check is performed")
+    val testProbe = TestProbe()
+    testProbe.send(checkActor, ExpungeOverdueLostTasksActor.Tick)
+    testProbe.receiveOne(3.seconds)
```

Comment:
ReceiveOne needs a timeout, adapted layout from other actor unit tests. What would you suggest?